tests/network/vmi_multus.go

```diff
@@ -1357,6 +1428,27 @@ var _ = SIGDescribe("Macvtap", func() {
 	})
 })
 
+func changeInterfaceMACAddress(vmi *v1.VirtualMachineInstance, interfaceName string, newMACAddress string) error {
+	const maxCommandTimeout = 5 * time.Second
+
+	commands := []string{
+		"ip link show",
```

Comment:
This one and the last one are leftovers, right?

Reply:
Right, they had been removed